packages/mui-joy/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@mui/joy",
-  "version": "5.0.0-beta.32",
+  "version": "5.0.0-beta-32",
```

```suggestion
  "version": "5.0.0-beta.32",
```
Please skip all the packages that are not stable.